app/controllers/api_controller/renderer.rb

```diff
@@ -457,7 +457,7 @@ def fetch_typed_subcollection_actions(method, is_subcollection)
       return unless is_subcollection
       ctype = @req[:collection].to_sym
       sakey = "#{@req[:subcollection]}_subcollection_actions".to_sym
-      collection_config.fetch_path(ctype, sakey, method.to_sym)
+      collection_config[ctype][sakey] && collection_config[ctype][sakey][method.to_sym]
```

Comment:
I considered this. As I see it, could go one of two ways:
* Monkey patch Config::Options (hesitant to do this)
* Make the Options object a private method of `Settings`, and use method missing to forward everything else to the Options. Hesitant to do this too though, since a Settings class would have many more inherited methods than an OpenStruct (which is what Options is). I'm not sure if this could be a real problem, but I can see the utility of adding this and probably more helper methods to the Settings object

Either way, it seemed sufficiently complicated to want to defer this to a follow up PR, if that's OK with you?